server/src/test/scala/org/http4s/server/middleware/ChunkAggregatorSpec.scala

```diff
@@ -27,10 +27,8 @@ class ChunkAggregatorSpec extends Http4sSpec {
     def checkResponse(body: EntityBody[IO], transferCodings: List[TransferCoding])(
         responseCheck: Response[IO] => MatchResult[Any]): MatchResult[Any] = {
       val service: HttpService[IO] = HttpService.lift[IO] { _ =>
-        Ok()
-          .putHeaders(`Transfer-Encoding`(NonEmptyList(TransferCoding.chunked, transferCodings)))
-          .removeHeader(`Content-Length`)
-          .withBody(body)
+        Ok(body, `Transfer-Encoding`(NonEmptyList(TransferCoding.chunked, transferCodings)))
```

Comment:
Sometimes I think `Message` should have a `.chunked`, which adds this header and removes any `Content-Length` header.  Having both is bad.

Reply:
👍 from me on `.chunked`